tests/test_spec.py

```diff
@@ -178,6 +178,29 @@ def test_get(val):
             "schema": {"$ref": "#/components/schemas/PaginationMetadata"},
         }
 
+    def test_all_headers_are_documented(self, app):
```

Comment:
I'm tempted to add a comment such as 

```py
# Non-regression test for https://github.com/marshmallow-code/flask-smorest/issues/578
```

to make the test intent more explicit.

Reply:
I did add a `#noqa: E501` to stop flake8 complaining about the length of that line.